docs/digger/admin/using-http-proxy.asciidoc

```diff
@@ -34,11 +35,12 @@ You can run AeroGear Digger behind a HTTP proxy server by defining the following
 You can define these variables either in an inventory file, or via CLI.
 Please refer to the http://docs.ansible.com/ansible/playbooks_variables.html[Ansible Playbook Variables Docs] for more details.
 
-NOTE: There are issues with using HTTP proxies that require authentication.
```

Comment:
* NOTE: aerogear digger does not currently support proxy that requires authentication

Reply:
@pmacko1 that sounds like a pre-req

Reply:
@finp it is already included in the upstream prereqs

Reply:
My suggestion above was more about the wording of this. Saying `There are issues with using HTTP proxy that requires authentication` is not clear on the fact that we want to say it is unsupported

Reply:
WARNING: HTTP Proxies that require authentication are not supported.

Reply:
resolved